Backend/request_handler/server.py

```diff
@@ -148,10 +160,6 @@ def get_prediction_status(file_id):
         else:
             return jsonify({'status': -1, 'error': 'Prediction data not found for the file ID'}), 404
 
-@app.route('/get_classes', methods=['GET'])
-def get_classlist():
-    return jsonify({'data': classifier.get_classlist()})
-
```

Comment:
I think you should ask the purpose of a module before removing it from the person who implemented this.
This was a enhancement feature for future that use may know what characters can be recognized by this app.